gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/BakeController.groovy

```diff
@@ -43,9 +44,17 @@ class BakeController {
     bakeService.bakeOptions(cloudProvider)
   }
 
+  @RequestMapping(value = "/api/v1/{region}/logs/{statusId}", method = RequestMethod.GET)
```

Comment:
Rosco ignores the region for this call. But it does define the argument on the interface to align with your internal bakery (and any other bakery somebody could potentially implement). Since it exists there, figure it made sense to define it here. Will also help with troubleshooting via logs I think.